test/functional/normalization/json_format_test.go

```diff
@@ -0,0 +1,52 @@
+package normalization
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/test/functional"
+	"github.com/openshift/cluster-logging-operator/test/helpers/types"
+	"github.com/openshift/cluster-logging-operator/test/matchers"
+	"strings"
+)
+
+var _ = Describe("[Functional][Normalization][Json] Parse json format log", func() {
+
+	var (
+		framework *functional.FluentdFunctionalFramework
+		// json message
+		json = "{\"name\":\"fred\",\"home\":\"bedrock\"}"
+		// Template expected as output Log
+		outputLogTemplate = functional.NewApplicationLogTempate()
+	)
+
+	BeforeEach(func() {
+
+		framework = functional.NewFluentdFunctionalFramework()
+		functional.NewClusterLogForwarderBuilder(framework.Forwarder).
+			FromInput(logging.InputNameApplication).
+			ToFluentForwardOutput()
+		Expect(framework.Deploy()).To(BeNil())
+		Expect(framework.WritesJsonApplicationLogs(json, 1)).To(BeNil())
+	})
+	AfterEach(func() {
+		framework.Cleanup()
+	})
+
+	Context("when parsing json logs as un-structured", func() {
+		It("should be logged as a message", func() {
+			outputLogTemplate.Message = strings.Replace(json, "\"", "", -1)
+			raw, err := framework.ReadApplicationLogsFrom("fluentforward")
```

Comment:
I believe you can use the api type `logging.OutputFluentForward`

Reply:
Done